PB Search ignores "sort" parameter

seancribbs opened the pull request:
`riak_search_pb_query` extracts the `sort` field from the incoming message but doesn't apply it to the query/result. See also http://lists.basho.com/pipermail/riak-users_lists.basho.com/2013-January/010700.html

Commit:
Sort the documents after performing the query (as the WM resource does it).

aurora commented:
i would like to ask, if there are any news regarding this. is there any roadmap, when this bug (including my patch) get's fixed?

engelsanchez committed:
Fix use of sort function

With this fix, the sort parameter seems to finally work through the PB
interface.

engelsanchez commented:
@seancribbs I stopped by to review this issue, but reproduced the problem @aurora saw. I just checked in a fix to the use of the sort function. I locally verified sort working through the PB interface with this.  I suppose now we have to find someone else to review this :)

seancribbs commented:
@engelsanchez Sounds good, should we get @rzezeski?

jonmeredith commented:
Focussing efforts on YZ.
